### DISCUSSION TITLE
run multiple jobs per ec2 (Runner Worker)

### DISCUSSION THREAD

**Discussion by lefterisALEX** [npalm/terraform-aws-gitlab-runner#1148] [General]
Hello,
I have the setup with docker-machine runner - one runner agent and i noticed that for each job a new runner worker is launched. That means i have too many ec2 instances launched when i execute multiple jobs in parallel.
How can i define how many jobs i want each runner  to pickup?

I tried this settgin the `runner_worker.request_concurrency` , but seems to not be what i am looking for.
```
    runner_worker = {
      ssm_access = true
      request_concurrency = 5
    }
```
and also `runner_worker_docker_machine_instance.max_growth_rate` , but if i understand correctly this sets how many runner workers are allowed to be launched in tota.l

**Comment by kayman-mk** [comment]
That is not supported by `docker+machine` as far as I know. See https://gitlab.com/gitlab-org/gitlab-runner/-/issues/29212 The issue was closed by GitLab as it is now part of the new Runner Autoscaling feature.

Seems that is will be solved with #1118
